src/index.js

```diff
@@ -8,6 +8,45 @@ const ALL_FOOD_ENDPOINT = 'filter.php?a=Jamaican';
 const ALL_FOOD_API_URL = FOOD_API_BASE_URL + ALL_FOOD_ENDPOINT;
 
 const foodListWrapper = document.getElementById('home');
+const commentPopup = document.getElementById('comment-popup');
+
+const displayPopUp = (id) => {
+  const URL = `${FOOD_API_BASE_URL}lookup.php?i=${id}`;
+  getData(URL).then((res) => {
+    const foodItem = res.meals[0];
+    commentPopup.classList.add('show');
```

Comment:
[OPTIONAL] I don't think it be noticeable but I think it is better to add show class when everything the popup is ready. I mean adding the class after filling it with content.

It wont be noticeable but logically we should show the popup when it is ready.

Reply:
- Makes sense now that you mention it. I'll be watchful next time. Thanks